handlers/payments.py

```
@@ -0,0 +1,231 @@
+    # Уведомляем пользователя
+    from main import bot
+    try:
+        await bot.send_message(
+            chat_id=payment['user_id'],
+            text=f"✅ **Пополнение подтверждено**\n\n"
+                 f"💰 Сумма: {payment['amount']} ₽\n"
+                 f"🆔 Заявка: #{payment_id}\n\n"
+                 f"Средства зачислены на ваш баланс!"
+        )
+    except:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)